docs/content/en/docs/getting-started/baremetal/bare-spec.md

```diff
@@ -577,294 +562,149 @@ spec:
     version: "0.1"
 ```
 
-### Bottlerocket TinkerbellTemplateConfig example
+## TinkerbellTemplateConfig fields
 
-Pay special attention to the `BOOTCONFIG_CONTENTS` environment section below if you wish to set up console redirection for the kernel and systemd.
-If you are only using a direct attached monitor as your primary display device, no additional configuration is needed here.
-However, if you need all boot output to be shown via a server's serial console for example, extra configuration should be provided inside `BOOTCONFIG_CONTENTS`.
+Each Action has its own set of environment variables that are used to configure the Action. EKS Anywhere embeds a set of images into HookOS. This means that when these images are used in an Action they will not be pulled from any external registry. No external network communication is needed to use them. These Actions are prefixed with `127.0.0.1/embedded`. 
```

Comment:
does user still have the option to pull images from public ecr like in the example before? If yes do we want to add an example of pulling from public ecr?

Reply:
Yes. Users can pull images from anywhere they want. I think if that wasn't clear that I should update the doc to make that more clear. In regard to providing an explicit example for pulling from ECR, I don't think we should. The whole idea of the embedded images is that pulling from ECR is not necessary. Also, there is not "latest" images in ECR for all the images. Each release we do has a new ECR location for images. Keeping these up to date in the docs is a lot of work and we have opted to not do this. This means that any exact reference provided in the docs is almost always out of date.